src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -119,7 +119,7 @@ internal static class LogMessages
         public const string IDX10610 = "IDX10610: Decryption failed. Could not create decryption provider. Key: '{0}', Algorithm: '{1}'.";
         public const string IDX10611 = "IDX10611: Decryption failed. Encryption is not supported for: Algorithm: '{0}', SecurityKey: '{1}'.";
         public const string IDX10612 = "IDX10612: Decryption failed. Header.Enc is null or empty, it must be specified.";
-        // public const string IDX10613 = "IDX10613:";
+        public const string IDX10613 = "IDX10613: Key unwrap failed using decryption Keys: '{0}'.\nExceptions caught:\n '{1}'.\ntoken: '{2}'.";
```

Comment:
You should use a new log id for this message. Log Id is considered deprecated if it's commented out. We are using this approach to make sure not to re-use a log id as it might lead to some confusion.

Reply:
That answers why they were commented out :) Thank you. I added a new message ID.